handlers_global.go

```diff
@@ -31,7 +32,7 @@ func handleProxy(p *Proxy) http.Handler {
 		}
 		// the server usually waits for this to return before finalizing the
 		// response, so this part must be done asynchronously
-		go p.ProxyMetrics(span.Attach(ctx), jsonMetrics, r.RemoteAddr)
+		go p.ProxyMetrics(span.Attach(ctx), jsonMetrics, strings.SplitN(r.RemoteAddr, ":", 2)[0])
```

Comment:
Can we insert a bounds check here? I think the http library is unlikely to break by returning an empty string, but on the off chance, I'd rather we send slightly-useless tags than breakage-loop all of our global metrics. :)

Reply:
> If s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s. If sep is empty, Split splits after each UTF-8 sequence. If both s and sep are empty, Split returns an empty slice.

Since we're splitting on a non-empty string literal, it's logically impossible to get a slice of length == 0!